src/xrpld/app/ledger/detail/LedgerMaster.cpp

```diff
@@ -34,10 +34,9 @@
 #include <xrpld/app/misc/TxQ.h>
 #include <xrpld/app/misc/ValidatorList.h>
 #include <xrpld/app/paths/PathRequests.h>
-#include <xrpld/app/rdb/backend/PostgresDatabase.h>
+#include <xrpld/app/rdb/RelationalDatabase.h>
```

Comment:
Is this include still needed?

Reply:
Yes, for type `LedgerHashPair`, which is a struct containing two `uint256`.